www/index.html

```diff
@@ -49,9 +84,24 @@
     <script src="js/homeCtrl.js"></script>
     <script src="js/settingsCtrl.js"></script>
     <script src="js/resourceService.js"></script>
-  </head>
-  <body ng-app="starter">
+    <script src="js/userInfoService.js"></script>
+</head>
+
+<body ng-app="starter" animation="slide-left-right-ios7">
+    <div>
```

Comment:
I got it from a template when i created the UI in ionic creator. I've removed them